sympy/parsing/autolev/test_examples/test10_out.py

```diff
@@ -0,0 +1,58 @@
+import sympy.physics.mechanics as me
+import sympy as sm
+import math as m
+import numpy as np
+
+frame_n=me.ReferenceFrame('n')
+frame_a=me.ReferenceFrame('a')
+frame_b=me.ReferenceFrame('b')
+particle_p=me.Particle('p', me.Point('p_pt'), None)
+particle_q=me.Particle('q', me.Point('q_pt'), None)
+point_o=me.Point('o')
+qa, qb = me.dynamicsymbols('qa qb')
+qad, qbd = me.dynamicsymbols('qa qb', 1)
+qad2, qbd2 = me.dynamicsymbols('qa qb', 2)
+la, lb=sm.symbols('la lb', real=True)
+k, lo=sm.symbols('k lo', real=True)
+g=sm.symbols('g', real=True)
+b, c=sm.symbols('b c', real=True)
+f1, f2 = me.dynamicsymbols('f1 f2')
+stretch = me.dynamicsymbols('stretch')
+stretchd = me.dynamicsymbols('stretch', 1)
+mp=sm.symbols('mp')
+particle_p.mass = mp
+mq=sm.symbols('mq')
+particle_q.mass = mq
+frame_a.orient(frame_n, 'Axis', [qa, frame_n.z])
+frame_b.orient(frame_n, 'Axis', [qb, frame_n.z])
+particle_p.point.set_pos(point_o, la*frame_a.x)
+particle_q.point.set_pos(particle_p.point, lb*frame_b.x)
+frame_a.set_ang_vel(frame_n, qad*frame_a.z)
+frame_b.set_ang_vel(frame_n, qbd*frame_b.z)
+point_o.set_vel(frame_n, 0)
+particle_p.point.v2pt_theory(point_o,frame_n,frame_a)
+particle_q.point.v2pt_theory(particle_p.point,frame_n,frame_b)
+loq = particle_q.point.pos_from(point_o).magnitude()
+stretch = loq-lo
+uvec=particle_q.point.pos_from(point_o)/loq
+stretchd = me.dot(particle_q.point.vel(frame_n), uvec)
+u_qad=me.dynamicsymbols('u_qad')
+u_qbd=me.dynamicsymbols('u_qbd')
+kd_eqs = [qad-u_qad, qbd-u_qbd]
+forceList = [(particle_p.point,particle_p.mass*(g*frame_n.x)), (particle_q.point,particle_q.mass*(g*frame_n.x)+ (-1*k*stretch-b*stretchd)*uvec), (point_o,-1*((-1*k*stretch-b*stretchd)*uvec) + f1*frame_n.x+f2*frame_n.y), (frame_a,-1*c*qad*frame_a.z), (frame_b,-1*c*qbd*frame_b.z)]
+kane = me.KanesMethod(frame_n, q_ind=[qa,qb], u_ind=[u_qad, u_qbd], kd_eqs = kd_eqs)
+fr, frstar = kane.kanes_equations([particle_p, particle_q], forceList)
+zero = fr+frstar
+zero = sm.Matrix([i.collect(g)for i in zero]).reshape((zero).shape[0], (zero).shape[1])
+
+import scipy.integrate as sc
+import pydy.codegen.ode_function_generators as pd
```

Comment:
This makes pydy a dependency of sympy, no? I don't think that we want any of the numerics based on pydy functions in sympy. SymPy is currently a strict dependency of PyDy, not the other way around. What you can do is generate numeric evaluation results using `evalf()`. You can compare the arbitrary precision values.

Reply:
I'm not sure what you mean by using evalf() here. Is it fine to use the other imports?
Is it fine to do it like you have done here : http://www.moorepants.info/blog/npendulum.html?
You don't seem to use a PyDy import here.

Reply:
If you want to check numerical results in SymPy you need to use things like `xreplace`, `subs`, and `evalf`. This calls the underlying mpmath library and outputs arbitrary precision results. For long complicated expressions, evaluating them in both Autolev and SymPy numerically for random inputs is a good way to check the correctness.

You can use lambdify as in the npendulum blog post too, but this will give you floating point precision.

Reply:
I haven't written tests to compare the numerical outputs of SymPy and Autolev yet.
The current tests in test_autolev.py and parsing/test_examples just compare the parser generated output against the correct output so that if anything is changed one can see if anything is broken.

The code generated here is the code for Simulation of Eoms after the system has been specified.